### ISSUE TITLE
Camera stop range being in half blocks not full blocks

### ISSUE THREAD

**Issue by Aid0nModder** [alesan99/mari0_ae#80]
Simple change

**Comment by alesan99** [comment]
I'm curious of the purpose of this. Maybe since the game is usually showing half blocks vertically?
I made it full blocks for convenience so you don't have to align it carefully.

**Comment by Aid0nModder** [comment]
Yeah, I find often I want to snap the screen to half a tile to mimic the border on the top or bottom of the screen, like hiding a secret area.
